docs/UserGuide.md

```diff
@@ -216,13 +211,13 @@ please refer to the [Features](#features) section below.
 
 ### <a name="dm">Data models</a>
 
-All tasks stored in TaskTracker will be automatically categorized into three different types: `floating task`, `deadline task`, and `event`, depending on the types and number of parameters entered when created.
+All tasks stored in TaskTracker will be automatically categorized into three different types: `floating task`, `deadline task`, and `event`, depending on the types of parameters entered when created.
 
 |A/An... | has...|
 |----| :-------------------------------------|
 |Floating Task | only a task name|
 |Deadline task |  end time and date |
-|Event | start time and date,  end time and date|
+|Event | start time and date, `to`, end time and date|
```

Comment:
I think I also revised a little bit of this part. And I have already created a pull request which has been approved already. Maybe revise this part again after my PR is merged.